client/scss/components/_progressbar.scss

```diff
@@ -23,3 +23,10 @@
     padding-inline-end: 1em;
   }
 }
+
+//Progress Bar for Windows high contrast mode
+@media (forced-colors: active) {
+  .progress .bar {
+    forced-color-adjust: none;
```

Comment:
Replace this line with `background-color: <system color you select>`

Reply:
I’ve gone ahead with this as `ButtonText`, after trying a few other options to no avail. `ButtonText` feels like safe bet since the text will always contrast with the background.

![progress-bar](https://user-images.githubusercontent.com/877585/178125305-9991896c-17b4-440a-89c8-32ee0994dc29.gif)